ui/react-ui/src/Pages/Users.js

```diff
@@ -0,0 +1,72 @@
+import React, {Component} from 'react'
+
+class Users extends Component{
+
+    constructor(props){
+      super(props)
+      this.state={
+        users:[],
+        isLoading:false,
+        isError: false
+      }
+    }
+    // Async function to get request 
+    async componentDidMount(){
+      this.setState({isLoading:true})
+      const response = await fetch("https://nate-temp-bucket.s3-us-west-2.amazonaws.com/nate_sample.json")
+      if(response.ok){
+        const users = await response.json()
+        console.log(users)
+        this.setState({users, isLoading:false})
+      }else{
+        this.setState({isError:true, isLoading:false})
+      }
+    }
+
+    renderTableHeader= () => {
+      return Object.keys(this.state.users[0]).map(attr=><th key = {attr}>
+        {attr.toUpperCase()}
+      </th>)
+      
+    }
+
+    renderTableRows = () =>{
+      return this.state.users.map(user=>{
+        return(
+          <tr key={user.userId}>
+            <td>{user.userId}</td>
+            <td>{user.firstName}</td>
+            <td>{user.lastName}</td>
+            <td>{user.email}</td>
+
+          </tr>
+        )
+      })
+    }
+
+    render(){
+      const {users, isLoading, isError} = this.state
+      if(isLoading){
+        return <div>Loading...</div>
+      }
+      
+      if(isError){
+        return <div>Error...</div>
+      }
+
+      return users.length > 0
+      ?(
+        <table>
+          <thead>
+            <tr>{this.renderTableHeader()}</tr>
+          </thead>
+          <tbody>{this.renderTableRows()}</tbody>
+        </table>
+
```

Comment:
Instead of making a table, maybe we can use an npm package like https://www.npmjs.com/package/react-json-to-table
![Screen Shot 2021-02-01 at 6 18 15 PM](https://user-images.githubusercontent.com/22699679/106543135-007fc400-64ba-11eb-8c98-ab207ab60ece.png)